app/src/main/java/org/schabi/newpipe/util/urlfinder/PatternsCompat.java

```diff
@@ -361,7 +361,7 @@
 
     public static final Pattern EMAIL_ADDRESS
             = Pattern.compile(
-            "[a-zA-Z0-9\\+\\.\\_\\%\\-\\+]{1,256}"
+            "[a-zA-Z0-9\\.\\_\\%\\-\\+]{1,256}"
                     + "\\@"
```

Comment:
Android Studio say that there are a lot more redundant escapes and that the field is unused. So we could even delete it.
``` java
 "[a-zA-Z0-9+._%\\-]{1,256}"
                    + "@"
```

Reply:
This class is copied from the AOSP. That's why I'd say we should not modify it at all. I removed the commit.